application/src/test/kotlin/com/studentcenter/weave/application/service/application/UserRefreshTokenApplicationServiceTest.kt

```diff
@@ -0,0 +1,83 @@
+package com.studentcenter.weave.application.service.application
+
+import com.studentcenter.weave.application.common.exception.AuthExceptionType
+import com.studentcenter.weave.application.common.properties.JwtTokenPropertiesFixtureFactory
+import com.studentcenter.weave.application.port.inbound.UserRefreshTokenUseCase
+import com.studentcenter.weave.application.port.outbound.UserRefreshTokenRepositorySpy
+import com.studentcenter.weave.application.port.outbound.UserRepositorySpy
+import com.studentcenter.weave.application.service.domain.impl.UserDomainServiceImpl
+import com.studentcenter.weave.application.service.util.impl.UserTokenServiceImpl
+import com.studentcenter.weave.application.service.util.impl.strategy.OpenIdTokenResolveStrategyFactoryStub
+import com.studentcenter.weave.domain.entity.User
+import com.studentcenter.weave.domain.entity.UserFixtureFactory
+import com.studentcenter.weave.support.common.exception.CustomException
+import io.kotest.assertions.throwables.shouldThrow
+import io.kotest.core.spec.DisplayName
+import io.kotest.core.spec.style.DescribeSpec
+import io.kotest.matchers.shouldBe
+import io.kotest.matchers.types.shouldBeInstanceOf
+import java.util.*
+
+@DisplayName("UserRefreshTokenApplicationService")
+class UserRefreshTokenApplicationServiceTest : DescribeSpec({
+
+    val userRepositorySpy = UserRepositorySpy()
+    val userRefreshTokenRepositorySpy = UserRefreshTokenRepositorySpy()
+    val userDomainServiceImpl = UserDomainServiceImpl(userRepositorySpy)
+    val userTokenServiceImpl = UserTokenServiceImpl(
+        jwtTokenProperties = JwtTokenPropertiesFixtureFactory.create(),
+        userRefreshTokenRepository = userRefreshTokenRepositorySpy,
+        openIdTokenResolveStrategyFactory = OpenIdTokenResolveStrategyFactoryStub(),
+    )
+    val sut = UserRefreshTokenApplicationService(
+        userDomainService = userDomainServiceImpl,
+        userTokenService = userTokenServiceImpl,
+        userRefreshTokenRepository = userRefreshTokenRepositorySpy,
+    )
+
+    afterEach {
+        userRepositorySpy.clear()
+        userRefreshTokenRepositorySpy.clear()
+    }
+
+    describe("유저 토큰 갱신 유스케이스") {
+        context("갱신 토큰이 만료되지 않았으면") {
```

Comment:
p4) 만료라는 용어보다 유효하다는 표현이 더 테스트에 적절한 것 같아요. 유효한 조건이 추가로 생겨서 추가할 수 있구요
```suggestion
        context("갱신 토큰이 유효하다면") {
```